beetsplug/ftintitle.py

```diff
@@ -110,6 +140,7 @@ def __init__(self) -> None:
                 "keep_in_artist": False,
                 "preserve_album_artist": True,
                 "custom_words": [],
+                "bracket_keywords": DEFAULT_BRACKET_KEYWORDS.copy(),
```

Comment:
Simply define them inline instead of using the constant - they aren't (or shouldn't be) used anywhere else.
```suggestion
                "bracket_keywords": [
                    ...
                ],
```